login-workflow/src/new-architecture/screens/CreatePasswordScreen/CreatePasswordScreen.tsx

```diff
@@ -19,14 +20,22 @@ export const CreatePasswordScreen: React.FC<CreatePasswordScreenProps> = (props)
     const confirmRef = useRef(null);
     const [passwordInput, setPasswordInput] = useState(password ?? '');
     const [confirmInput, setConfirmInput] = useState(confirmPassword ?? '');
+    const [isLoading, setIsLoading] = useState(false);
     const passwordRequirements = defaultPasswordRequirements(t);
 
-    const onNext = useCallback(() => {
-        nextScreen({
-            screenId: 'CreatePassword',
-            values: { password: passwordInput, confirmPassword: confirmInput },
-        });
-    }, [confirmInput, passwordInput, nextScreen]);
+    const onNext = useCallback(async (): Promise<void> => {
+        setIsLoading(true);
```

Comment:
We should move the loader logic inside try block, because we are calling async action inside that.

Reply:
Done. Please review again